types/p-event/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for p-event 1.3
+// Project: https://github.com/sindresorhus/p-event#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export = pEvent;
+
+declare function pEvent<T, TRest = T>(emitter: pEvent.Emitter<T, TRest>, event: string | symbol, options: MultiArgsOptions): Promise<Array<T | TRest>>;
+declare function pEvent<T>(emitter: pEvent.Emitter<T>, event: string | symbol, options?: pEvent.Options): Promise<T>;
+
+declare namespace pEvent {
+    interface Emitter<T = any, TRest = T> {
+        on?: AddRmListenerFn<T, TRest>;
```

Comment:
I also thought the same way first. But then, look at the node's EventEmitter: it has an `on()` but no `off()`. So strictly speaking, I'd have to permute all possible combinations of `on()` and `off()`-like interfaces here. I thought it would be overkill. Correct me if I'm wrong.